### ISSUE TITLE
Error with inserting images to PDF with Markdown (Origin: bugzilla #719541)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#5366]
status NEW severity *normal* in component *general* for *---*
Reported in version *1.8.5* on platform *Other*
Assigned to: Dimitri van Heesch

On 2013-11-29 11:36:59 +0000, Sergey Dryakhlov wrote:

    I use Markdown for generating documentation in pdf and html. In order to
    include image  I use syntax discribed in the chapter "Marhdown
    support/Standard Markdown/Images":
    ![Caption text](/path/to/img.jpg)
    It works correctly for html-documentation, however it does not work for pdf.

On 2015-07-10 11:42:54 +0000, N wrote:

    Images in Markdownfiles are still ignored in Latex-Output.
    ==========================================================
    
    While Doxygens own Syntax (`@image latex Picture.png "Bild3"
    width=\textwidth`) works fine the Markdownsyntax (`![Bild](Picture.png)`) is
    ignored in latexoutput.
    
    Below you will find an example.
    
    
    About my system:
    ----------------
    
    * Linux-Mint 17.1
    * Doxygen
        * version 1.8.6 (installed from the usual trusty/main-Repo)
        * I tried version 1.8.11 from the git-repo too (same problem)
        * Doxygen's default configuration
    
    
    Example
    -------
    
    ````
    ### An example markdownfile with images ###
    
    This will work for html but NOT work for LaTeX (but should):
    
    ![Bild](Picture.png)
    
    
    But this works fine:
    
    @image html Picture.png "Bild2"
    @image latex Picture.png "Bild3" width=\textwidth
    
    End of example
    ````
    
    
    See also
    --------
    
    * Bug # 683112 - markdown: images are not included in latex

**Comment by albert-github** [comment]
Code has been integrated in master on github.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.16.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
